spec/solver_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../solver'
+
+describe Solver do
+  before(:each) do
+    @solver = Solver.new
+  end
+
+  context 'When testing the factorial() method' do
+    it 'should return 1 for the factorial of 0' do
+      result = @solver.factorial(0)
+      expect(result).to eq 1
+    end
+    it 'should return 362,880 for the factorial of 9' do
+      result = @solver.factorial(9)
+      expect(result).to eq 362_880
+    end
+    it 'should return 3,628,800 for the factorial of 10' do
+      result = @solver.factorial(10)
+      expect(result).to eq 3_628_800
+    end
+  end
+
+  context 'Checking if a string is reverse string' do
+    it 'should return olleh for the string hello' do
+      result = @solver.reverse_string('hello')
+      expect(result).to eq 'olleh'
+    end
+
+    it 'should return anihc for the string china' do
+      result = @solver.reverse_string('china')
+      expect(result).to eq 'anihc'
+    end
+  end
+
+  context 'Checking a number and return FizzBuzz logic string' do
+    it 'should return Fizz if n is divisible by 3' do
+      result = @solver.fizzbuzz(3)
+      expect(result).to eq 'Fizz'
+    end
+
+    it 'should return Buzz if divisible by 5' do
+      result = @solver.fizzbuzz(5)
+      expect(result).to eq 'Buzz'
+    end
+
+    it 'should return FizzBuzz if divisible by 3 and 5 at the same time' do
+      result = @solver.fizzbuzz(15)
+      expect(result).to eq 'FizzBuzz'
+    end
+  end
```

Comment:
- [ ] For the FizzBuzz testing you need to include a scenario for when the value is neither divisible by 3 nor 5. For example, if the number 7 is supplied it should return the string value: '7'.

Reply:
well resolved! ✔️

Reply:
Hey @rloterh and @julie-ify,
I just pushed the changes, they resolved the missing scenarios.
Thanks.